1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,31 @@
 function highlightWords(paragraph, colours) {
   // Write your code here...
+  let content = document.getElementById("content");
+  let pElement=document.createElement("p");
+  let selectElemet= document.createElement("select");
+  for (let color of colours){
+  let optionElement=document.createElement("option");
+  optionElement.innerText= color
+  optionElement.value= color 
+  selectElemet.appendChild(optionElement)
+  }
```

Comment:
This is nice - simply following the instructions or task description is a good way to tack a piece of homework. What about layout though.  It's good to ensure indentation to help others looking at your code. Don't forget semicolons too.